internal/database/repository/partitions.go

```diff
@@ -54,19 +57,18 @@ func (s *PostgresRepository) UpsertPartitions(ctx context.Context, partitions []
 	return nil
 }
 
-func (s *PostgresRepository) GetAllPartitions(ctx context.Context) ([]*dao.PartitionInfo, error) {
+func (s *PostgresRepository) GetAllPartitions(ctx context.Context) ([]*model.PartitionInfo, error) {
```

Comment:
Here, you should probably get only active partitions. We don't need to get deleted partitions to operate on them.
We should also change the name, so we know that we are fetching active partitions.

Reply:
Hi, Thanks for the awesome reviews 🚀 
I found that, `GetAllPartitions` is also being used by the `webservice` [here](https://github.com/G-Research/yunikorn-history-server/blob/04d00cef9427529ca5e84809be925f076ead5058/internal/webservice/routes.go#L227). I think frontend might need the deleted partition data as well to show some graphical design on them. Therefore, I updated the logic in `findPartitionDeleteCandidates()`